crates/stackable-telemetry/src/instrumentation/axum/mod.rs

```diff
@@ -363,9 +428,12 @@ impl SpanExt for Span {
         // Setting server.address and server.port
         // See https://opentelemetry.io/docs/specs/semconv/http/http-spans/#setting-serveraddress-and-serverport-attributes
 
-        if let Some(server_socket_address) = req.server_socket_address() {
-            span.record("server.address", server_socket_address.ip().to_string())
-                .record("server.port", server_socket_address.port());
+        if let Ok((host, port)) = req.server_host() {
+            // NOTE (@Techassi): We cast to i64, because otherwise the field
+            // will NOT be recorded as a number but as a string. This is likely
+            // an issue in the tracing-opentelemetry crate.
+            span.record("server.address", host)
+                .record("server.port", port as i64);
```

Comment:
**note (non-blocker)**: As discussed via DM, we will not use the Semantic Convention consts in the `record` functions until they can be used in the `Span` definitions (and `instrument(fields(...))` too.